app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryUpdate.kt

```diff
@@ -28,7 +28,7 @@ class CustomDrugEntryUpdate : Update<CustomDrugEntryModel, CustomDrugEntryEvent,
       is PrescribedDrugFetched -> prescriptionFetched(model, event.prescription)
       is RemoveDrugButtonClicked -> {
         val update = model.openAs as OpenAs.Update
-        dispatch(RemoveDrugFromPrescription(update.prescribedDrugUuid))
+        next(model.drugInfoProgressStateLoading(), RemoveDrugFromPrescription(update.prescribedDrugUuid))
```

Comment:
Can you check with the design to see if we show the progress for the entire sheet when the remove button is clicked?

Reply:
I checked they asked to add the progress state for add button inside the button([slack message](https://simpledotorg.slack.com/archives/CFK2PHJ1L/p1631172421170700?thread_ts=1630316031.043300&cid=CFK2PHJ1L)). I will raise another pr for it.